src/App.jsx

```diff
@@ -14,33 +20,96 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getPreparedGoods(goods, sortField, query) {
+  const preparedGoods = [...goods];
+  const SORT_FIELD_REVERSE = query;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SORT_FIELD_LENGTH:
+          return good1.length - good2.length;
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+        case SORT_FIELD_ALPHABETICALLY:
+          return good1.localeCompare(good2);
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (SORT_FIELD_REVERSE) {
```

Comment:
The condition `if (SORT_FIELD_REVERSE)` is misleading because `SORT_FIELD_REVERSE` is not a constant but a dynamic value based on the `query` parameter. Ensure that the logic here correctly checks whether the goods should be reversed based on the `query` parameter, which is actually a boolean indicating if the list should be reversed.